lib/pdf/generator.rb

```diff
@@ -0,0 +1,68 @@
+class Pdf::Generator
+
+  include MarkdownHelper
+
+  def from_markdown(text)
+    pdf = Prawn::Document.new
+    html_document = Nokogiri::HTML(markdown(text))
+
+    font_path = Rails.root.join("app", "assets", "fonts")
+    pdf.font_families.update(
+      "Roboto" => {
+        normal: "#{font_path.to_s}/Roboto-Light.ttf",
+        italic: "#{font_path.to_s}/Roboto-LightItalic.ttf",
+        bold: "#{font_path.to_s}/Roboto-Bold.ttf",
+      }
+    )
+
+    pdf.font "Roboto"
+
+    pdf.repeat(:all) do
+      y_position = pdf.cursor
+      pdf.text_box "Projeto de iniciativa popular", size: 12, style: :bold, at: [70, pdf.cursor]
```

Comment:
Could you please give names to what you are trying to accomplish here? Say a method named `build_header`, `draw_footer` or things like that.